test/recipes/80-test_cms.t

```diff
@@ -509,6 +509,50 @@ subtest "CMS <=> CMS consistency tests, modified key parameters\n" => sub {
     }
 };
 
+# Returns the number of matches of a Content Type Attribute in a binary file.
+sub contentType_matches {
+  # Read in a binary file
+  my ($in) = @_;
+  open (HEX_IN, "$in") or die("open failed for $in : $!");
+  binmode(HEX_IN);
+  local $/;
+  my $str = <HEX_IN>;
+
+  #Find ASN1 data for a Content Type Attribute (with a OID od PKCS7 data)
```

Comment:
"with a. OID __od__ PKCS7 data" ?

Reply:
'd' is right next to 'f' on the keyboard.